controllers/utils.go

```diff
@@ -0,0 +1,28 @@
+package controllers
+
+import (
+	"context"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	// DataPlaneFinalizer is the finalizer which indicates that an object needs to
+	// have its configuration removed from the dataplane before it can be deleted.
+	DataPlaneFinalizer = "blixt/dataplane-configuration"
+)
+
+func isDataPlaneFinalizerSet(obj client.Object) bool {
```

Comment:
`controllerutil.ContainsFinalizer`  from "sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 Has this already done for yah :)

Reply:
d989894183724bbd39b163dcb68984dd77d4111c adds this :+1: